BCStudentSoftwareDevTeam/celts
# 1145 email minor interest

## Issue
Have an "Email All Interested" button to allow the admin to send a message to each student who has expressed interest in the CCE Minor. Resolves #1145

## Solution
Make a button that emails everyone who's inside the interested students table. 
We did remove the ability to show how many people are in that table to put the button there (see image). 
### Before
![image](https://github.com/BCStudentSoftwareDevTeam/celts/assets/89316750/e9396328-e3d1-46a3-a74d-bec0ecc3b216)
### After
![image](https://github.com/BCStudentSoftwareDevTeam/celts/assets/89316750/9ff03949-1d52-4e77-8a3c-77633f95af1e)
Let us know if this is a problem and we'll figure out something that keeps the functionality and is cosmetically awesome.

## To test
- Navigate to the minor management page under Admin in the sidebar.
![image](https://github.com/BCStudentSoftwareDevTeam/celts/assets/89316750/655dee80-bbff-471d-9570-0bb48e3a4318)
- Click the email all button by the interested students table and ensure that a window opens with your system's default email application
- Ensure that the email addresses from the table match the users in the table.
